### ISSUE TITLE
player.cpp

### ISSUE THREAD

**Issue by AJenbo** [diasurgical/devil-nightly#109]
- [ ] `44A8E6` (`j_player_cpp_init`)
   - [ ] `44A8EB` (`player_cpp_init`)
   - [ ] `44A8F6` (`SetPlayerGPtrs`)
   - [ ] `44A911` (`LoadPlrGFX`)
   - [x] `44AB70` (`InitPlayerGFX`)
   - [ ] `44ABB4` (`InitPlrGFXMem`)
   - [ ] `44ADC8` (`GetPlrGFXSize`)
   - [x] `44AE89` (`FreePlayerGFX`)
   - [x] `44AF37` (`NewPlrAnim`)
   - [x] `44AF9C` (`ClearPlrPVars`)
   - [x] `44AFED` (`SetPlrAnims`)
   - [x] `44B1FD` (`ClearPlrRVars`)
   - [ ] `44B274` (`CreatePlayer`)
   - [x] `44B582` (`CalcStatDiff`)
   - [x] `44B5C3` (`NextPlrLevel`)
   - [x] `44B6C8` (`AddPlrExperience`)
   - [x] `44B7F8` (`AddPlrMonstExper`)
   - [ ] `44B83C` (`InitPlayer`)
   - [x] `44BB33` (`InitMultiView`)
   - [ ] `44BB6D` (`CheckLeighSolid`)
   - [x] `44BCC2` (`SolidLoc`)
   - [ ] `44BCEB` (`PlrDirOK`)
   - [ ] `44BD9A` (`PlrClrTrans`)
   - [ ] `44BDDD` (`PlrDoTrans`)
   - [x] `44BE5E` (`SetPlayerOld`)
   - [ ] `44BE95` (`FixPlayerLocation`)
   - [x] `44BF2D` (`StartStand`)
   - [ ] `44BFE8` (`StartWalkStand`)
   - [ ] `44C070` (`PM_ChangeLightOff`)
   - [ ] `44C13D` (`PM_ChangeOffset`)
   - [ ] `44C1E2` (`StartWalk`)
   - [ ] `44C3AC` (`StartWalk2`)
   - [ ] `44C5CF` (`StartWalk3`)
   - [x] `44C81E` (`StartAttack`)
   - [x] `44C8BB` (`StartRangeAttack`)
   - [x] `44C973` (`StartPlrBlock`)
   - [x] `44CA26` (`StartSpell`)
   - [x] `44CB95` (`FixPlrWalkTags`)
   - [ ] `44CC62` (`RemovePlrFromMap`)
   - [ ] `44CCD8` (`StartPlrHit`)
   - [ ] `44CDFD` (`DestroyDupeItem`)
   - [ ] `44CEC9` (`StartPlayerKill`)
   - [ ] `44D1F4` (`PlrDeadItem`)
   - [ ] `44D2F3` (`DropHalfPlayersGold`)
   - [ ] `44D70B` (`SyncPlrKill`)
   - [ ] `44D79B` (`j_StartPlayerKill`)
   - [ ] `44D7A0` (`RemovePlrMissiles`)
   - [x] `44D8D1` (`InitLevelChange`)
   - [ ] `44D973` (`StartNewLvl`)
   - [x] `44DA6F` (`RestartTownLvl`)
   - [ ] `44DAFC` (`StartWarpLvl`)
   - [x] `44DB74` (`PM_DoStand`)
   - [x] `44DB77` (`PM_DoWalk`)
   - [x] `44DCE5` (`PM_DoWalk2`)
   - [x] `44DE30` (`PM_DoWalk3`)
   - [ ] `44DFB1` (`WeaponDur`)
   - [ ] `44E0BC` (`PlrHitMonst`)
   - [ ] `44E442` (`PlrHitPlr`)
   - [ ] `44E669` (`PlrHitObj`)
   - [ ] `44E6A6` (`PM_DoAttack`)
   - [x] `44E8B8` (`PM_DoRangeAttack`)
   - [x] `44E9AC` (`ShieldDur`)
   - [x] `44EA4D` (`PM_DoBlock`)
   - [x] `44EAC6` (`PM_DoSpell`)
   - [x] `44EC06` (`PM_DoGotHit`)
   - [ ] `44ECBC` (`ArmorDur`)
   - [ ] `44ED7B` (`PM_DoDeath`)
   - [ ] `44EE22` (`CheckNewPath`)
   - [ ] `44F9BA` (`PlrDeathModeOK`)
   - [ ] `44F9FC` (`ValidatePlayer`)
   - [ ] `44FB32` (`ProcessPlayers`)
   - [x] `44FD31` (`CheckCheatStats`)
   - [x] `44FD8A` (`ClrPlrPath`)
   - [ ] `44FDBA` (`PosOkPlayer`)
   - [ ] `44FE9E` (`MakePlrPath`)
   - [ ] `44FF6F` (`CheckPlrSpell`)
   - [ ] `450217` (`SyncPlrAnim`)
   - [ ] `45036D` (`SyncInitPlrPos`)
   - [x] `4504E4` (`SyncInitPlr`)
   - [ ] `450508` (`CheckStats`)
   - [ ] `450621` (`ModifyPlrStr`)
   - [ ] `4506DB` (`ModifyPlrMag`)
   - [ ] `450788` (`ModifyPlrDex`)
   - [ ] `45082C` (`ModifyPlrVit`)
   - [x] `4508CF` (`SetPlayerHitPoints`)
   - [x] `45091E` (`SetPlrStr`)
   - [x] `450993` (`SetPlrMag`)
   - [x] `4509DF` (`SetPlrDex`)
   - [x] `450A54` (`SetPlrVit`)
   - [x] `450AA0` (`InitDungMsgs`)
   - [ ] `450AC4` (`PlayDungMsgs`)

**Comment by mewmew** [comment]
> @mewmew some renames in devilution:
> CheckLeighSolid -> InitPlayerLoc
> DestroyDupeItem -> RespawnDeadItem

Good catch! Fixed in rev diasurgical/scalpel@ded10fb19cf6f51d5ba7595fa9af5e2146589e83.

**Comment by AJenbo** [comment]
@mewmew i have marked j_player_cpp_init and player_cpp_init as they are not recognized by Devilution-comparer and also j_player_cpp_init doesn't exist in the codebase (magic?).

**Comment by seritools** [comment]
Either cvdump.exe or the linker mangle the names of those to `$1` to `$5`. You can find all removed/missing defined functions over [here](https://github.com/diasurgical/devilution-comparer/blob/master/removed-functions.toml).
